### PR TITLE
Feature/migration

### PR DESCRIPTION
**Pull request by alextremp**
This PR includes:

* The OpenAds demo project as it was included in the OpenAds repository
* It's still using the 2.2.0 version of OpenAds until the 3 version is released to use the connector dependencies

### CODE REVIEW

**Review by rafamarquezv — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@schibstedspain/openads-demo",
+  "version": "2.2.0",
```
Comment (victuxbb) [inline package.json]: version 1.0.0 ?

**Review by victuxbb — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (alextremp) [reply package.json]: I live in the future

Comment (alextremp) [reply package.json]: done @victuxbb :)

**Review by victuxbb — APPROVED** [review]

**Pull request merged**